packages/peregrine/lib/talons/SearchPage/useSearchPage.js

```diff
@@ -224,7 +244,33 @@ export const useSearchPage = (props = {}) => {
 
     useScrollTopOnChange(currentPage);
 
+    const sortingMethods = sortData
+        ? sortData?.products?.sort_fields?.options
```

Comment:
Same thing about optional chaining here

Reply:
Same as before too. The tests for this were breaking until I added them.